pkg/aspect/listChecker/types.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package listChecker
+
+import (
+	"istio.io/mixer/pkg/aspect"
+	"github.com/golang/protobuf/proto"
+)
+
+type (
+	// AdapterConfig -- Adapter Author visible typed config for
+	// creating new listChecker aspect
+	// This may or may not have listcheckerpb.Config
```

Comment:
I don't understand this "may or may not" sentence, and I don't see where this config would be in the first place.